src/sbt-test/jdkpackager/test-package-mappings/build.sbt

```diff
@@ -24,36 +24,36 @@ Universal / mappings ++= {
 }
 
 lazy val iconGlob = sys.props("os.name").toLowerCase match {
-  case os if os.contains("mac") ⇒ "*.icns"
-  case os if os.contains("win") ⇒ "*.ico"
-  case _ ⇒ "*.png"
+  case os if os.contains("mac") => "*.icns"
+  case os if os.contains("win") => "*.ico"
+  case _                        => "*.png"
 }
 
 jdkAppIcon := (baseDirectory.value / ".." / ".." / ".." / ".." / "test-project-jdkpackager" ** iconGlob).getPaths.headOption
   .map(file)
 
 TaskKey[Unit]("checkImage") := {
   val (extension, os) = sys.props("os.name").toLowerCase match {
-    case osys if osys.contains("mac") ⇒ (".app", 'mac)
-    case osys if osys.contains("win") ⇒ (".exe", 'windows)
-    case _ ⇒ ("", 'linux)
+    case osys if osys.contains("mac") => (".app", 'mac)
+    case osys if osys.contains("win") => (".exe", 'windows)
+    case _                            => ("", 'linux)
   }
   val expectedImage = (JDKPackager / target).value / "bundles" / (name.value + extension)
   println(s"Checking for '${expectedImage.getAbsolutePath}'")
   assert(expectedImage.exists, s"Expected image file to be found at '$expectedImage'")
 
   val files = os match {
-    case 'mac ⇒
+    case 'mac =>
```

Comment:
Speaking of deprecated features, I see there are symbol literals in the codebase